### PR TITLE
Uglify now outputs error in Browserify/Uglify/Sourcemap recipe

### PR DESCRIPTION
**Pull request by bhough**
Added error output for Uglify, so that it doesn't just fail silently.

### CODE REVIEW

**Comment by bhough** [comment]
This addresses: https://github.com/gulpjs/gulp/issues/990

File: docs/recipes/browserify-uglify-sourcemap.md [hunk 1]
```diff
@@ -24,7 +25,7 @@ gulp.task('javascript', function () {
     .pipe(browserified)
     .pipe(sourcemaps.init({loadMaps: true}))
         // Add transformation tasks to the pipeline here.
-        .pipe(uglify())
+        .pipe(uglify().on('error', gutil.log))
```
Comment (callumacrae) [inline docs/recipes/browserify-uglify-sourcemap.md]: You can just put this on the line below!

```
.pipe(uglify())
  .on('error'...
```

**Comment by bhough** [comment]
Updated! Thanks @callumacrae silly oversight on my part.

**Comment by terinjokes** [comment]
You may also be interested in linking to [Combining streams to handle errors](https://github.com/gulpjs/gulp/blob/master/docs/recipes/combining-streams-to-handle-errors.md) for it's suggestion for handling errors.

**Comment by phated** [comment]
@terinjokes I've added the note to the recipe.

Thanks @bhough!

**Pull request merged**